lametro/static/css/city_custom.css

```diff
@@ -128,6 +128,32 @@ div#google_translate_element {
 
 #ceo-table td { vertical-align: middle; }
 
+/* Styling for autocomplete, default from library */
+.autocomplete-suggestions {
+  border: 1px solid #999;
+  background: #FFF;
+  overflow: auto;
+}
+
+.autocomplete-suggestion {
+  padding: 2px 5px;
+  white-space: nowrap;
+  overflow: hidden;
+}
+
+.autocomplete-selected { background: #F0F0F0; }
+
+.autocomplete-suggestions strong {
+  font-weight: normal;
+  color: #3399FF;
```

Comment:
the default blue is a little obtrusive – how about we highlight the matching portion with the same salmon/red color we use for page links/the search button?